jdaviz/core/loaders/resolvers/resolver.py

```diff
@@ -0,0 +1,232 @@
+import warnings
+from traitlets import Bool, List, Unicode, observe
+
+from jdaviz.core.template_mixin import PluginTemplateMixin, SelectPluginComponent, with_spinner
+from jdaviz.core.registries import loader_parser_registry, loader_importer_registry
+from jdaviz.core.user_api import LoaderUserApi
+
+__all__ = ['BaseResolver']
+
+
+class FormatSelect(SelectPluginComponent):
+    def __init__(self, plugin, items, selected, default_mode='first'):
+        """
```

Comment:
Please move the class docstring to after `class FormatSelect` but before `def __init__`. Also start it with a concise description of the purpose of this class, to be consistent with other API docs.